.travis.yml

```diff
@@ -14,3 +14,7 @@ env:
 script: bundle exec ./go ci_build
 notifications:
   webhooks: http://project-monitor.cf.18f.us/projects/hub/status
+
+script: bundle exec rake run_about_yml_check
```

Comment:
I'm happy to do that, but was hoping to have an example for other teams who
wanted to automatically run this check on their own files. Is there a way
to preserve this diff in GitHub so that I can link it from the about_yml
README?

On Tue, Dec 29, 2015 at 11:27 AM, Mike Bland <notifications@github.com>
wrote:

> In .travis.yml <https://github.com/18F/hub/pull/503#discussion_r48551366>:
>
> > @@ -14,3 +14,7 @@ env:
> >  script: bundle exec ./go ci_build
> >  notifications:
> >    webhooks: http://project-monitor.cf.18f.us/projects/hub/status
> > +
> > +script: bundle exec rake run_about_yml_check
>
> I think this should get added to the ./go ci_build command instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/hub/pull/503/files#r48551366>.
>